src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_IsPatternOperator.cs

```diff
@@ -192,8 +199,6 @@ private void LowerOneTest(BoundDagTest test, bool invert = false)
                 _factory.Syntax = test.Syntax;
                 switch (test)
                 {
-                    case BoundDagAssignmentEvaluation:
```

Comment:
>case BoundDagAssignmentEvaluation: [](http://example.com/codeflow?start=20&length=34)

Would it make sense to keep this case with throw?

Reply:
LowerEvaluation is the very next thing we call with an explicit unreachable case mentioned above.